e2e/test/scenarios/models/model-actions.cy.spec.js

```diff
@@ -174,12 +174,6 @@ describe(
       cy.findByText("New").click();
       popover().findByText("Action").click();
 
-      cy.findByText("Select a database").click();
-      popover().within(() => {
-        cy.findByText("Sample Database").should("not.exist");
-        cy.findByText("QA Postgres12").click();
-      });
-
```

Comment:
This is no longer needed because the database picker would automatically select the only available option. Before, the picker would have the sample database in the list of databases, but now it gets filtered out at the parent component because actions are not disabled for it